eng/pipelines/runtime-staging.yml

```diff
@@ -273,6 +273,191 @@ jobs:
           creator: dotnet-bot
           testRunNamePrefixSuffix: Mono_$(_BuildConfig)
 
+#
+# Build the whole product using Mono and run libraries tests, for Wasm.Build.Tests
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - Browser_wasm_win
+    variables:
+      # map dependencies variables to local variables
+      - name: wasmbuildtestsContainsChange
+        value: $[ dependencies.evaluate_paths.outputs['SetPathVars_wasmbuildtests.containsChange'] ]
+    jobParameters:
+      isExtraPlatforms: ${{ variables.isExtraPlatformsBuild }}
+      testGroup: innerloop
+      nameSuffix: WasmBuildTests
+      buildArgs: -s mono+libs+host+packs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true /p:TestWasmBuildTests=true /p:TestAssemblies=false /p:BrowserHost=$(_hostedOs)
+      timeoutInMinutes: 180
+      condition: >-
+        or(
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_wasmbuildtests.containsChange'], true),
+          eq(variables['isRollingBuild'], true))
```

Comment:
Should we run this only on PRs? Rolling build is already covered by `extra-pipelines`

Reply:
Do you mean `path-change && on-pr`?
Is there any existing variable for checking if it's a PR?

```yml
 and(
          eq(dependencies.evaluate_paths.outputs['SetPathVars_wasmbuildtests.containsChange'], true),
          eq(variables['Build.Reason'], 'PullRequest'))
```

Reply:
Just remove the `IsRollingBuild` condition and the or(). The path-change will never be true if the build is a rolling build.